modules/serverless-quarkus-functions-overriding-liveness-readiness.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies
+//
+// * functions/reference/serverless-developing-quarkus-functions.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-quarkus-functions-overriding-liveness-readiness_{context}"]
+= Overriding liveness and readiness probe values
+
+You can override `liveness` and `readiness` probe values for your Quarkus functions. This allows you to modify the timing of health checks performed on the function.
```

Comment:
WDYM by timing?

Reply:
@matejvasek Good catch, I did `s/modify the timing of/configure/` in all three sections.